templates/staging/ia-archiver-cronjob.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ia-import
```

Comment:
I honestly didn't put too much thought into the namespace. I think it's in here because I was following some documentation (could be wrong there), but it could be anything.

Reply:
Ok. I think we should move this to production, then.